packages/tldraw/src/lib/utils/export/exportAs.ts

```diff
@@ -1,28 +1,62 @@
 import { Editor, sanitizeId, TLFrameShape, TLImageExportOptions, TLShapeId } from '@tldraw/editor'
-import { exportToBlob } from './export'
 
 /** @public */
-export type TLExportType = 'svg' | 'png' | 'jpeg' | 'webp' | 'json'
+export type TLExportType = 'svg' | 'png' | 'jpeg' | 'webp'
+
+/** @public */
+export interface ExportAsOptions extends TLImageExportOptions {
+	format: TLExportType
+	/** Name of the exported file. If undefined a predefined name, based on the selection, will be used. */
+	name?: string
+}
 
 /**
  * Export the given shapes as files.
  *
  * @param editor - The editor instance.
  * @param ids - The ids of the shapes to export.
- * @param format - The format to export as.
- * @param name - Name of the exported file. If undefined a predefined name, based on the selection, will be used.
  * @param opts - Options for the export.
  *
  * @public
  */
 export async function exportAs(
 	editor: Editor,
 	ids: TLShapeId[],
-	format: TLExportType = 'png',
-	name: string | undefined,
-	opts: TLImageExportOptions = {}
+	opts: TLImageExportOptions & { format: TLExportType; name?: string }
```

Comment:
wait, why does this have `& { format: TLExportType }` that's already part of TLImageExportOptions

Reply:
oh i see - gotcha